tests/util_test.go

```diff
@@ -36,13 +38,65 @@ type ClientTests struct {
 	fixture  *Fixture
 }
 
+type OrderedMap struct {
+	Order []string
+	Map   map[string]interface{}
+}
+
+func (om *OrderedMap) UnmarshalJSON(b []byte) error {
```

Comment:
Please leave a comment as to why we are unmarshaling and remarshaling above each of these two functions.

Also, see my other comments. I think this only works at the root level. Nested objects are still out of order. If there isn't a way to fix them all, then I'd prefer we don't fix half of them only.

Reply:
These two functions only work with non-array object strings. 

For example, it works when the JSON string is `{id:"xxx", address:"xxx"}` but not in `[{id:"xxx"}, {id:"xxx"}].

Basically, it will not work when the API endpoint return an array object string which happens in `TestCarrierAccountAll`

Reply:
It looks like json.Marshal should be able to handle arrays as well as dictionaries, or am I interpreting this wrong? https://golang.cafe/blog/golang-json-marshal-example.html

Reply:
Yes, `json.Marshal` can handle both but it will sort the items in alphabetical order which is not what we wanted.

Reply:
We have decided to remove this function and keep the order alphabetical